package.json

```diff
@@ -89,60 +89,60 @@
     }
   },
   "dependencies": {
-    "chalk": "^2.3.2",
+    "chalk": "^2.4.1",
     "cross-spawn": "^6.0.5",
     "diff": "^3.5.0",
     "enhanced-resolve": "^4.0.0",
-    "envinfo": "^5.4.0",
+    "envinfo": "^5.7.0",
     "glob-all": "^3.1.0",
     "global-modules": "^1.0.0",
-    "got": "^8.2.0",
+    "got": "^8.3.1",
     "import-local": "^1.0.0",
-    "inquirer": "^5.1.0",
-    "interpret": "^1.0.4",
+    "inquirer": "^5.2.0",
+    "interpret": "^1.1.0",
     "jscodeshift": "^0.5.0",
-    "listr": "^0.14.0",
+    "listr": "^0.14.1",
     "loader-utils": "^1.1.0",
-    "lodash": "^4.17.5",
+    "lodash": "^4.17.10",
     "log-symbols": "^2.2.0",
     "mkdirp": "^0.5.1",
     "p-each-series": "^1.0.0",
     "p-lazy": "^1.0.0",
-    "prettier": "^1.5.3",
-    "supports-color": "^5.3.0",
+    "prettier": "^1.12.1",
+    "supports-color": "^5.4.0",
     "v8-compile-cache": "^2.0.0",
     "webpack-addons": "^1.1.5",
     "yargs": "^11.1.0",
-    "yeoman-environment": "^2.0.0",
-    "yeoman-generator": "^2.0.4"
+    "yeoman-environment": "^2.1.1",
+    "yeoman-generator": "^2.0.5"
   },
   "peerDependencies": {
     "webpack": "^4.0.0"
   },
   "devDependencies": {
-    "@commitlint/cli": "^6.1.3",
+    "@commitlint/cli": "^6.2.0",
     "@commitlint/prompt-cli": "^6.1.3",
-    "@commitlint/travis-cli": "^6.1.3",
+    "@commitlint/travis-cli": "^6.2.0",
     "bundlesize": "^0.17.0",
-    "codecov": "^3.0.0",
+    "codecov": "^3.0.2",
     "commitizen": "^2.9.6",
-    "conventional-changelog-cli": "^1.3.17",
+    "conventional-changelog-cli": "^1.3.22",
     "conventional-changelog-lint-config-cz": "^0.3.0",
     "cz-customizable": "^5.2.0",
     "eslint": "^4.19.1",
     "eslint-plugin-node": "^6.0.1",
     "husky": "^0.14.3",
-    "jest": "^22.4.3",
-    "jest-cli": "^22.4.3",
+    "jest": "^22.4.4",
+    "jest-cli": "^22.4.4",
     "jsdoc": "^3.5.5",
-    "lint-staged": "^7.0.0",
-    "nyc": "^11.6.0",
+    "lint-staged": "^7.1.2",
+    "nyc": "^11.8.0",
     "prettier-eslint-cli": "^4.7.1",
     "schema-utils": "^0.4.5",
-    "semantic-release": "^15.1.8",
-    "webpack": "^4.2.0",
-    "webpack-dev-server": "^3.0.0",
-    "travis-deploy-once": "^5.0.0"
+    "semantic-release": "^15.5.0",
+    "travis-deploy-once": "^5.0.0",
+    "webpack": "^4.8.3",
+    "webpack-dev-server": "^3.1.4"
```

Comment:
The diff looks messy here because of wrong ordering in the previous version of the code. :grin: